playbook.md

```diff
@@ -30,7 +30,7 @@ We think that it's very important to have a talented team if we're going to succ
 We are helpful, useful and expert. We give practical and pragmatic advice to each other, and to our clients.
 
 #### Honest
-We are honest, trustworthy and straightforward. We give plain-spoken, frank, accurate feedback and advice, and we never mislead or obfuscate.
+We are honest, trustworthy and straightforward. We give plain-spoken, frank, accurate feedback and advice, and we never mislead or wilfully confuse.
```

Comment:
To me, part of the meaning of `obfuscate` is clouding the issue by bombarding someone with information - to blind with science, in other words. So that's what I was trying to capture with `wilfully confuse`.